app/src/main/java/org/mozilla/fenix/tabtray/FenixTabsAdapter.kt

```diff
@@ -67,10 +67,30 @@ class FenixTabsAdapter(
 
     override fun onBindViewHolder(holder: TabViewHolder, position: Int) {
         super.onBindViewHolder(holder, position)
-        val newIndex = tabCount - position - 1
-        (holder as TabTrayViewHolder).updateAccessibilityRowInfo(
+
+        val isListTabView = context.settings().listTabView
+
+        val itemIndex = if (isListTabView) {
+            tabCount - position - 1
+        } else {
+            position
+        }
+
+        val rowIndex = if (isListTabView) {
+            itemIndex
+        } else {
+            itemIndex / 2
```

Comment:
Does this have to be 2? Looking below it sounds like it could be more but I might be misunderstanding?